api/property_service.py

```diff
@@ -44,58 +44,58 @@ def dehexify(hex_str):
-      "propertyName": dehexify( mpData['name'] ), 
-      "propertySubcategory": dehexify( mpData['subcategory'] ), 
-      "propertyUrl": dehexify( mpData['url'] ), 
-      "propertyType": '0002' if mpData['divisible'] == True else '0001', 
-      "formatted_property_type": int('0002' if mpData['divisible'] == True else '0001'), 
-      "transactionType": txData[3], 
-      "transactionVersion": txData[4], 
-      "tx_hash": txData[0], 
+      "propertyCategory": dehexify( mpData['category'] ),
+      "propertyData": dehexify( mpData['data'] ),
+      "propertyName": dehexify( mpData['name'] ),
+      "propertySubcategory": dehexify( mpData['subcategory'] ),
+      "propertyUrl": dehexify( mpData['url'] ),
+      "propertyType": '0002' if mpData['divisible'] == True else '0001',
+      "formatted_property_type": int('0002' if mpData['divisible'] == True else '0001'),
+      "transactionType": txData[3],
+      "transactionVersion": txData[4],
+      "tx_hash": txData[0],
       "tx_time": txJson['blocktime']
     }
 
-    if txType == 50: ret['numberOfProperties'] = mpData['totaltokens']; 
-    
+    if txType == 50: ret['numberOfProperties'] = mpData['totaltokens'];
+
     if txType == 51:
-      ret['numberOfProperties'] = mpData['tokensperunit']; 
+      ret['numberOfProperties'] = mpData['tokensperunit'];
       ret['currencyIdentifierDesired'] = mpData['propertyiddesired']
       ret['deadline'] = mpData['deadline']
       ret['earlybirdBonus'] = mpData['earlybonus']
       ret['percentageForIssuer'] = mpData['percenttoissuer']
-      
+
     if txType == 54:
-      ret['numberOfProperties'] = mpData['totaltokens']; 
+      ret['numberOfProperties'] = mpData['totaltokens'];
       ret['issuances'] = mpData['issuances']
 
     #Fields that didn't make it, may become relevant at a later time
 
     #"tx_type_str": "Fundraiser property creation"
-    #"formatted_amount": 0, 
-    #"formatted_ecosystem": 1, 
-    #"formatted_previous_property_id": 0, 
-    #"formatted_transactionType": 51, 
-    #"formatted_transactionVersion": 0, 
-    #"baseCoin": "00", 
-    #"color": "bgc-new", 
-    #"dataSequenceNum": "01", 
-    #"details": "unknown", 
-    #"icon": "unknown", 
-    #"index": "215", 
-    #"invalid": false, 
-    #"method": "multisig", 
-    #"to_address": "1ARjWDkZ7kT9fwjPrjcQyvbXDkEySzKHwu", 
-    #"tx_method_str": "multisig", 
+    #"formatted_amount": 0,
+    #"formatted_ecosystem": 1,
+    #"formatted_previous_property_id": 0,
+    #"formatted_transactionType": 51,
+    #"formatted_transactionVersion": 0,
+    #"baseCoin": "00",
+    #"color": "bgc-new",
+    #"dataSequenceNum": "01",
+    #"details": "unknown",
+    #"icon": "unknown",
+    #"index": "215",
+    #"invalid": false,
+    #"method": "multisig",
+    #"to_address": "1ARjWDkZ7kT9fwjPrjcQyvbXDkEySzKHwu",
+    #"tx_method_str": "multisig",
     #"update_fs": false
 
 
-    return json.dumps([ ret ]) #only send back mapped schema 
```

Comment:
That is the only call in the wallet but probably best to leave it as an array if only because it might be used outside of our platform. But i agree for standardizing with proper json response outputs on any new or v2 api's going forward